packages/xo-web/src/icons.scss

```diff
@@ -405,6 +405,10 @@
       @extend .fa;
       @extend .fa-list-alt;
     }
+    &-backup {
```

Comment:
It's already defined [here](https://github.com/vatesfr/xen-orchestra/pull/4257/files#diff-f1fbfb3950088945c9e0fe78c4d2ed3fR278).